Bio/SeqUtils/MeltingTemp.py

```diff
@@ -759,7 +773,13 @@ def Tm_GC(
     """
     if saltcorr == 5:
         raise ValueError("salt-correction method 5 not applicable to Tm_GC")
-    seq = str(seq)
+    if isinstance(seq, SeqRecord):
+        seq = str(seq.seq)
+    elif isinstance(seq, Seq._SeqAbstractBaseClass):
+        seq = str(seq)
```

Comment:
You can call `bytes` on `Seq` objects and `SeqRecord` objects.

Reply:
Calling `bytes` is faster as it avoids converting the `bytes` object to a string.

Reply:
See https://github.com/biopython/biopython/pull/4875#issuecomment-2440177361